game/scripts/npc/abilities/troll_warlord_fervor.txt

```diff
@@ -21,12 +21,12 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "attack_speed"                                    "5 10 15 20 25 30" //OAA
+        "attack_speed"                                    "15 20 25 30 35 40"
```

Comment:
I discussed this with you already. I don't like this. Max attack speed he should get from this ability should be 450. This includes the talent (+3 stacks). I can do the math for you but 450 is very fair if you ask me.

Reply:
Its now 9 stacks at all levels. Max attack speed with talent is 480.